src/App.js

```diff
@@ -2,10 +2,25 @@
 import './App.css';
 import React from 'react';
 import Calculator from './component/calculator';
+import Quote from './component/fetch-data';
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
```

Comment:
- [ ]  It would be better for your App.js and Calculator.js to be functional components, functional components are the modern ways to write components and in previous milestones you implemented this functionality please update your class components to functional components 👍🏿

Reply:
Thanks! I have refactored my code to functional component.